### PR TITLE
Update for flutter 1.2.1

### PR DESCRIPTION
**Pull request by zemanux**

### CODE REVIEW

**Comment by jsa-nic** [comment]
Note this fixes a compile issue using the latest Flutter versions. The issue is detailed in this ticket https://github.com/flutter/flutter/issues/26655 and appears as the following error: 

```
Compiler message:
.../Pub/Cache/hosted/pub.dartlang.org/painter-0.3.1/lib/painter.dart:167:20: Error: A value of type 'Future<Image>' can't be assigned to a variable of type 'Image'.
 - 'Future' is from 'dart:async'.
 - 'Image' is from 'dart:ui'.
Try changing the type of the left hand side, or casting the right hand side to 'Image'.
    return picture.toImage(width, height);
                   ^
```

File: lib/painter.dart [hunk 1]
```diff
@@ -163,12 +163,13 @@ class PictureDetails{
 
   const PictureDetails(this.picture,this.width,this.height);
 
-  Image toImage(){
+  Future<Image> toImage(){
```
Comment (diegomxc) [inline lib/painter.dart:166]: Works¡¡, Thx

File: lib/painter.dart [hunk 2]
```diff
@@ -163,12 +163,13 @@ class PictureDetails{
...
     return picture.toImage(width, height);
   }
 
   Future<Uint8List> toPNG() async{
-    return (await toImage().toByteData(format: ImageByteFormat.png)).buffer.asUint8List();
+    final image = await toImage();
```
Comment (diegomxc) [inline lib/painter.dart:171]: good

**Comment by freshteapot** [comment]
What more needs to be done to this, to get this merged in? :).

**Pull request merged**